hs_collection_resource/views.py

```diff
@@ -27,6 +29,11 @@ def update_collection(request, shortkey, *args, **kwargs):
     msg = ""
     metadata_status = "Insufficient to make public"
     new_coverage_list = []
+
+    hasPart = "hasPart"
+    site_url = current_site_url()
+    relation_value_template = site_url + "/resource/{0}/"
```

Comment:
Since this line of code seems to be used in many places, may be you can define this variable in utils and then access it wherever needed.

```ACION_VALUE_TEMPLATE = current_site_url() +  "/resource/{0}/"```